app/src/app/close.js

```diff
@@ -49,8 +52,16 @@ export default function(e) {
     }
     if (isOSX()) app.hide()
   }
+  const lastUrl = state.mainWindow.webContents.getURL()
+  const isLastUrlChat = (
+    parseUrl(lastUrl)
+      .pathname
+      .split('/')
+      .filter(p => p !== '')[0]
+  ) === chatPath
+
   storage.set('lastUrl', {
-    url: state.mainWindow.webContents.getURL(),
+    url: isLastUrlChat ? lastUrl : false,
```

Comment:
why to set anything at all if last url is not chat? also I assume it should be null